test/integration/bundle/suite.go

```diff
@@ -447,6 +447,41 @@ var _ = Describe("Integration", func() {
 		}, eventuallyTimeout, eventuallyPollInterval).Should(BeNil(), "checking that bundle was re-added to newly labelled namespace")
 	})
 
+	Context("Reconcile consistency", func() {
+		It("should have stable resourceVersion", func() {
+			var configMap corev1.ConfigMap
+			Expect(cl.Get(ctx, client.ObjectKey{Namespace: "kube-system", Name: testBundle.Name}, &configMap)).To(Succeed())
+			resourceVersion := configMap.ResourceVersion
+			Consistently(komega.Object(&configMap)).Should(HaveField("ObjectMeta.ResourceVersion", Equal(resourceVersion)))
+		})
+
+		It("should have stable resourceVersion for JKS target", func() {
+			Expect(komega.Update(testBundle, func() {
+				testBundle.Spec.Target.AdditionalFormats = &trustapi.AdditionalFormats{
+					JKS: &trustapi.JKS{KeySelector: trustapi.KeySelector{Key: "target.jks"}}}
+			})()).To(Succeed())
+
+			configMap := &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Namespace: "kube-system", Name: testBundle.Name}}
+			Eventually(komega.Object(configMap)).Should(HaveField("BinaryData", HaveKey("target.jks")))
+
+			resourceVersion := configMap.ResourceVersion
+			Consistently(komega.Object(configMap)).Should(HaveField("ObjectMeta.ResourceVersion", Equal(resourceVersion)))
+		})
+
+		It("should have stable resourceVersion for PKCS12 target", func() {
```

Comment:
I really like these tests, feel free to add these to other projects if you have some extra time 😄.

Reply:
> I really like these tests, feel free to also add these to other projects if you have some extra time 😄.

Thanks, I also like them. Reconcile-loop errors like this one can be very annoying, and I have been looking for a good way of testing/detecting them.... I'll see what I have time for. 😉